Add comment section

NoyFlat opened the pull request:
Add gson dependency to pom.xml
Add form in html
Add function to fetch comments in script.js
Add java class Comment
Edit DataServlet.java to recive the comment from user and get the JSON from server

NoyFlat committed:
Add comment section
Add gson dependency to pom.xml
Add form in html
Add function to fetch comments in script.js
Add java class Comment
Edit DataServlet.java to recive the comment from user and get the JSON from server

orarch commented on portfolio/pom.xml:
```diff
@@ -35,7 +40,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Remove this TODO, you did it as part of this change :)

NoyFlat replied:
done

orarch commented on portfolio/src/main/java/com/google/sps/data/Comment.java:
```diff
@@ -0,0 +1,40 @@
+// limitations under the License.
+
+package com.google.sps.data;
+
```
Add class doc

NoyFlat replied:
done

orarch commented on portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:
```diff
@@ -14,19 +14,49 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```
Wildcard imports are bad:

https://rules.sonarsource.com/java/RSPEC-2208

NoyFlat replied:
done

NoyFlat committed:
Fix according to Or's comments
Remove TODO comments that aren't relevent
Add class doc to Comment.java
Remove wildcard import
Change variable names to be more informative

NoyFlat committed:
Add spacing

orarch reviewed (approved):
Nice job!
